jsk_footstep_msgs/FootstepArray input
---
jsk_footstep_msgs/FootstepArray output
